docs/concepts/models.md

```diff
@@ -11,20 +11,33 @@ Models share many similarities with Python's dataclasses, but have been designed
 differences that streamline certain workflows related to validation, serialization, and JSON schema generation.
 You can find more discussion of this in the [Dataclasses](dataclasses.md) section of the docs.
 
-Untrusted data can be passed to a model and, after parsing and validation, Pydantic guarantees that the fields
+Untrusted data can be passed to a model and, after validation, Pydantic guarantees that the fields
 of the resultant model instance will conform to the field types defined on the model.
 
 !!! note
-    Pydantic is primarily a parsing and transformation library, **not a validation library**.
-    Validation is a means to an end: building a model which conforms to the types and constraints provided.
+    The use of the terms "parse" and "validate" within Pydantic's context has been a source of confusion.
```

Comment:
I don't think we need the big about "parse" and "validate", instead we should just explain why we use the word "validate".